management/src/pages/RejectedphotoManagement.js

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from "react";
+import ImageGallery from 'react-image-gallery';
+import 'react-image-gallery/styles/css/image-gallery.css';
+import { Buffer } from 'buffer';
+
+export default function RejectedPhotoManagement() {
+    const images = [];
+    const [galleryImages, setGalleryImages] = useState(images);
+    const [successMessage, setSuccessMessage] = useState('');
+    {
+        successMessage && (
+            <div className="alert alert-success" role="alert">
+                {successMessage}
+            </div>
+        )
+    }
+
+    useEffect(() => {
+        fetchUserImages();
+
+        refreshData();
+    }, [galleryImages]);
+
+    const refreshData = () => 
+    {
+        if (galleryImages) {
+            galleryImages.forEach(i => {
+                if (i.approvalStatus === 'declined') {
+                    const data = `data:${i.image.contentType};base64,${Buffer.from(i.image.data)}`;
+                    images.push({
+                        original: data,
+                        thumbnail: data,
+                        srcSet: data,
+                        originalAlt: i.image.name,
+                        thumbnailAlt: i.image.name,
```

Comment:
```suggestion
                        thumbnailAlt: i.image?.name,
```
voor het geval image niet bestaat, voorkomt een runtime error